debian/rules

```diff
@@ -43,15 +43,15 @@ binary-syncd-rpc:
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
+    ifeq ($(shell cat /tmp/syncd-build), --enable-rpcserver=yes)
```

Comment:
It shouldn't matter, as my scripts are not in a debian build directory, so they will not get cleaned up after the change. Do you have other concerns here I'm missing?